src/components/postgresConnect.ts

```diff
@@ -476,6 +476,32 @@ class PostgresDB {
     }
   }
 
+  public getReviewCurrentStatusHistory = async (reviewId: number) => {
+    const text = `SELECT id, status, id FROM
+      review_status_history WHERE
+      review_id = $1 and is_current = true;`
+    try {
+      const result = await this.query({ text, values: [reviewId] })
+      return result.rows[0]
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  }
+
+  public addNewReviewStatusHistory = async (reviewId: number, status = 'Draft') => {
```

Comment:
It would be better to use the graphql type here instead. so we keep it up-to-date with the one in the database - in case we change it later.

Reply:
You mean the `status` type? There's a problem with that, which I also faced when you suggested the same thing on the Application Status update: GraphQL converts enums to ALLCAPS, so 'Draft' is not valid, it's expecting 'DRAFT'. But postGres expects 'Draft'. Not sure there's any way round this, unless we can somehow export Postgres types directly?

Reply:
We'd need something like this, I think: https://github.com/adelsz/pgtyped

Might be worth the effort?

Reply:
Hmm, a bit skeptical atm, of how much we want to typescript. I'll try to check out that package as suggested (if time allows atm).

Reply:
Yeah, definitely not a priority. Could be useful when we change some enums, then Typescript would quickly point out inconsistencies.

Reply:
Oh, that's not nice. I thought it would work the same way as we have on the front-end code.
Well, let's leave it for now then...